content/docs/concepts/errors.md

```diff
@@ -3,61 +3,62 @@ title: Errors
 weight: 3
 ---
 
-We leverage a concept of `positive`, `negative`, and `neutral` for success, errors, and no change or neutral states.
-These classes affect Pills and Text only.
+There are time you just want a small reflection of a greater state; good, bad, indifferent. More on the SPA/SWA side of development, we found use for small indications of that greater state and made it easier to leverage in Platform UI. 
 
+`positive`, `negative`, and `neutral` became a simple language to convery these. They currently affect [pills](/docs/components/pills/), [text](link here), and [backgrounds](link here), and may make their way further into our framework in the future. 
 
 ## Text
 
 <p class="text--positive">Positive</p>
 <p class="text--negative">Negative</p>
 <p class="text--neutral">Neutral</p>
 
-<div class="mt-3 mb-4">
 {{< highlight html >}}
 <p class="text--positive">Positive</p>
 <p class="text--negative">Negative</p>
 <p class="text--neutral">Neutral</p>
 {{< /highlight >}}
-</div>
-
 
 ## Pills
 
-<div class="pill pill--circle-empty positive"></div>
-<div class="pill pill--circle-empty negative"></div>
-<div class="pill pill--circle-empty neutral"></div>
-<div class="pill pill--circle-empty-dot positive"></div>
-<div class="pill pill--circle-empty-dot negative"></div>
-<div class="pill pill--circle-empty-dot neutral"></div>
-<br><br>
-<div class="pill positive">
-  Positive
+<div  class="pill pill--circle-empty positive"></div>
```

Comment:
These extra spaces seem weird. Is that like a Markdown thing or something?

Reply:
yes, they do.

Reply:
<img width="186" alt="Screen Shot 2022-04-11 at 1 17 01 PM" src="https://user-images.githubusercontent.com/6231956/162794553-64ab069e-c1fb-49b1-bec3-22a45e6513f5.png">